gdsa 1424 fix expired token issue on gdsdelete (#1210)
neo4j/graph-data-science-client: changelog.md

## Bug fixes

> * Requests to the Aura API that are rejected as unauthorized are now retried once with a newly minted OAuth token, instead of failing permanently while the expired token stays cached. This affected operations such as `GdsSessions.delete`, which could leave a session running.

Reviewer: ```suggestion
* Requests to the Aura API that are rejected as unauthorized are now retried once with a new OAuth token, instead of failing permanently while the expired token stays cached.
```
Let's remove some of those common LLM words :D Also we usually don't specify the failure scenarios in the bug fix announcements
